test/Microsoft.AspNetCore.Mvc.Core.Test/Assets/NonUtf8EncodedRequest.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
.cs file is encoded in utf8 so I guess it's not an option. What I wanted to do is to install System.Text.Encoding.CodePages and try to generate non utf8 string via code. That would allow to get rid of this file and be more reliable. Can I install such package? Can you add it to NetCore MyGet mirror?